Examples/Examples/BasemapGalleryExampleView.swift

```diff
@@ -34,15 +32,35 @@ struct BasemapGalleryExampleView: View {
     private let basemaps = initialBasemaps()
     
     var body: some View {
-        MapView(map: dataModel.map, viewpoint: initialViewpoint)
-            .overlay(alignment: .topTrailing) {
-                if showBasemapGallery {
-                    BasemapGallery(items: basemaps, geoModel: dataModel.map)
-                        .style(.automatic())
-                        .padding()
-                }
+        MapView(map: map, viewpoint: initialViewpoint)
+            
+            // Display in a sheet
+            //
+            .sheet(isPresented: $showBasemapGallery) {
+                BasemapGallery(items: basemaps, geoModel: map)
+                    .style(.grid)
```

Comment:
`.style(.grid)` is here to prevent the list style from being auto-selected on iPhone's, which would consume the entire screen width.

![0](https://user-images.githubusercontent.com/16397058/228953234-956e6c76-1776-461b-af25-fd71b526fbf1.png)